README.md

```diff
@@ -1,98 +1,187 @@
+- Generate multiple maps (and reports) automatically using QGIS Atlas and QGIS Reports
+- Flexible output direct to printer, or as image (raster), PDF, or SVG for further customization 
+- On-the-fly rendering enhancements using geometry generators (e.g. create and style new geometries from existing features)
+- Preview modes for inclusive map making (e.g. monochrome, color blindness)
+
+*[Example: Map of Bogota, Columbia in the style of Starry Starry Night, by Andrés Felipe Lancheros Sánchez](https://flic.kr/p/2jFfGJP)*
+
+![Map of Bogota, Columbia in the style of Starry Starry Night](https://live.staticflickr.com/65535/50327326323_3da28f0d86_b.jpg "Map of Bogota, Columbia in the style of Starry Starry Night")
+
+For more maps created with QGIS, visit the [QGIS Map Showcase Flickr Group](https://www.flickr.com/groups/2244553@N22/pool/with/50355460063/).
+
+![QGIS Map Showcase](images/README-md/qgis_map_showcase.png "QGIS Map Showcase")
+
+### 3. Advanced and robust geospatial analysis
+- Powerful processing framework with 200+ native processing algorithms
+- Access to 1000+ processing algorithms via providers such as GDAL, SAGA, GRASS, OrfeoToolbox, as well as custom models and processing scripts
+- Geospatial database engine (filters, joins, relations, forms, etc.), as close to datasource- and format-independent as possible 
+- Immediate visualization of geospatial query and geoprocessing results
+- Graphical modeler and batch processing
+
+*Example: Travel isochrones*
+
+![Example: Travel isochrones](images/README-md/network_analysis_2.png "Travel isochrones")
+
+*Example: Graphical model*
+
+![Example: graphical model](https://docs.qgis.org/latest/en/_images/models_model.png "Graphical model")
+
+### 4. Powerful customization and extensibility
+
+- Fully customizable user experience, including user interface and application settings that cater to power-users and beginners alike
+- Rich [expression engine](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/expression.html) for maximum flexibility in visualization and processing
+- Broad and varied [plugin ecosystem](https://plugins.qgis.org/) that includes data connectors, digitizing aids, advanced analysis and charting tools, in-the-field data capture, etc.
+- Style manager for creating, storing, and managing styles
+- [QGIS style hub](https://plugins.qgis.org/styles/) for easy sharing of styles
+- Python and C++ API for standalone (headless) applications as well as in-application comprehensive scripting (PyQGIS)
+
+*Example: Style manager*
+
+![Example: Style manager](https://docs.qgis.org/latest/en/_images/stylemanager.png "Style Manager")
+
+*Example: Plugins*
+
+![Example: Plugins](images/README-md/plugins_1.png "Plugins")
+
+<!-- Kill this one for now, since it's Python2 not 3
+Example: Python console
+
+![Example: Python console](https://docs.qgis.org/latest/en/_images/python_console_editor.png "Python console")
```

Comment:
we'll be happy to get this fixed in the docs :smile:

Reply:
:) -- though not sure we need to include it here anyway. Think best to limit to 2 examples per section, and python code is python code. We threw this one off the island because of the 2 vs 3, but think we ended up with a better set of 2 complementary ex anyway.